internal/msgs/en_config_descriptions.go

```diff
@@ -38,4 +38,6 @@ var (
 	ConfigEventsCatchupThreshold      = ffc("config.connector.events.catchupThreshold", "How many blocks behind the chain head an event stream or listener must be on startup, to enter catchup mode", i18n.IntType)
 	ConfigEventsCheckpointBlockGap    = ffc("config.connector.events.checkpointBlockGap", "The number of blocks at the head of the chain that should be considered unstable (could be dropped from the canonical chain after a re-org). Unless events with a full set of confirmations are detected, the restart checkpoint will this many blocks behind the chain head.", i18n.IntType)
 	ConfigEventsFilterPollingInterval = ffc("config.connector.events.filterPollingInterval", "The interval between polling calls to a filter, when checking for newly arrived events", i18n.TimeDurationType)
+	ConfigTxCacheSize                 = ffc("config.connector.txCacheSize", "Maximum of transactions to hold in the transaction info cache", i18n.IntType)
+	ConfigTxCacheTTL                  = ffc("config.connector.txCacheTTL", "Time to live for the transaction info cache", i18n.TimeDurationType)
```

Comment:
Not used?

Reply:
Can probably prune `ConfigBlockCacheTTL` too

Reply:
ah thought I removed, thank you